docs/spelling_wordlist.txt

```diff
@@ -359,7 +359,6 @@ DagRunState
 DAGs
 Dask
 dask
```

Comment:
can we remove these too?

Reply:
I am not sure if we can even remove daskexecutor to be honest (due to release notes),

Reply:
Yeah. We can't even remove `daskexecutor` because of release notes.